src/main/scala/org/ergoplatform/nodeView/wallet/requests/AssetIssueRequest.scala

```diff
@@ -52,6 +52,6 @@ class AssetIssueRequestDecoder(settings: ErgoSettings) extends Decoder[AssetIssu
       decimals <- cursor.downField("decimals").as[Int]
       feeOpt <- cursor.downField("fee").as[Option[Long]]
     } yield AssetIssueRequest(address, amount, name, description, decimals,
-      feeOpt.getOrElse(settings.walletSettings.defaultTransactionFee))
```

Comment:
Why?

Reply:
@catena2w, because when deserializing request as part of `RequestsHolder` we always have `request.fee = None` in each request, so this part automatically sets it to `settings.walletSettings.defaultTransactionFee` - it's wrong, as fee is set in special field of `RequestsHolder`

Reply:
Fixed in https://github.com/ergoplatform/ergo/pull/495/commits/7edc699ac3b37564fa270a7bc6944154e7014b45